### ISSUE TITLE
Update spelling on contact page

### ISSUE THREAD

**Issue by llienfec** [18F/fec-cms#1245]
User pointed out that "teletypewriter" is misspelled on the [Contact](https://www.fec.gov/contact-us/) page.

![image](https://user-images.githubusercontent.com/22914750/30340518-c693a2da-97c0-11e7-8b43-c0bc3b6db945.png)

cc: @johnnyporkchops

**Comment by AmyKort** [comment]
Is the going in the upcoming release?

**Comment by patphongs** [comment]
@AmyKort It is not in the current release. However, it's a simple fix that I could get merged in.

**Comment by AmyKort** [comment]
Thanks for taking care of this!

**Comment by jenniferthibault** [comment]
@patphongs I’m a little confused because this should have been taken care of already by my PR #1250 linked above. Any idea why hats not showing up?

**Comment by patphongs** [comment]
Good catch @jenniferthibault! Your PR was actually to an older release branch (release/public-20170913) that we never deployed due to all the circle issues we had. Then when we finally got things going, we had to close the old release branch since it was too old and create a new one release/public-20171011. So your change got lost in the shuffle. But I am glad that you brought this up because we should also check what other PRs directly to the old release branch got lost too! Will add a ticket to address this.
